client/client.go

```
@@ -22,16 +22,16 @@ var returnerror ltm.ErrorLTM
 func GTMWipDelete(c *gin.Context) {
 	var wipdelete gtm.RemoveWip
 
-	f5url, _ := gtm.Trafficmanager(c.Params.ByName("trafficmanager"))
+	f5url, err := gtm.Trafficmanager(c.Params.ByName("trafficmanager"))
+	if err != nil {
+		glog.Errorf("%s", err)
```

Do you want to return the error instead of only logging? Same below.